docs/griptape-cloud/data-sources/create-data-source.md

```diff
@@ -0,0 +1,9 @@
+# Data Sources
+
+Use [Data Sources](https://cloud.griptape.ai/data-sources) to connect and ingest your own data for use with your AI applications via [Knowledge Bases](../knowledge-bases/create-knowledge-base.md).
+
+## Create a Data Source
+
+You can [Create a Data Source](https://cloud.griptape.ai/data-sources/create) by specifying the required configuration for your chosen Data Source.
+
+If you do not see a data source configuration you'd wish to use, you can submit a request via [Discord](https://discord.gg/gnWRz88eym) or `hello@griptape.ai`.
```

Comment:
Are we going to always capitalize resources like data source?

Reply:
It was a standard we started in the framework docs that I carried through here. Missed a few though, thanks for the catch.